include/NAS2D/Resources/FontInfo.h

```diff
@@ -15,13 +15,15 @@
 
 struct GlyphMetrics
 {
-	GlyphMetrics() : uvX(0.0f), uvY(0.0f), uvW(0.0f), uvH(0.0f), minX(0), minY(0), maxX(0), maxY(0), advance(0) {}
```

Comment:
Hmm, this one is a struct. Having a constructor makes it non-aggregate. I'm uncertain if having default initialized values does the same. That may depend on the C++ language standard.

We should give some consideration as to whether or not default initialization is even useful here.

Reply:
Could you clarify?

Reply:
Reference: [aggregate initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization)

From C++11 until C++14, an aggregate is defined as not having default member initializers.

Since we use C++17, this should be fine.

Reply:
To be clear, this one is a bit of an aside. It's not really about the change, but about future potential.

By removing the constructor and replacing it with default member initialization, this type is now an aggregate (in C++17). That's a bonus.

Aggregate types relate to what is safe to `memcpy` around, or serialize directly to/from a file.

Reply:
Small observation: The `GlyphMetrics` struct is fully initialized in the only method that creates it. It's not all done at once though, so that's not super clear.

Full initialization tends to be typical of structs. I find it makes zero initialization a bit pointless. What I'd really want is a warning if a field is left uninitialized. There are cases where the compiler can warn about use of uninitialized fields, though in general, it can't warn for every possible case. Of course, it can only do that if the field is actually left uninitialized. That may be a consideration for leaving struct fields uninitialized by default.

At any rate, this is a positive change which clearly moves in a good direction. Default initialization is better than constructor list initialization. Whether we want any initialization at all is a separate question.